WordPress/src/test/java/org/wordpress/android/ui/domains/management/usedomain/UseDomainViewModelTest.kt

```diff
@@ -0,0 +1,85 @@
+package org.wordpress.android.ui.domains.management.usedomain
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.toList
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.test.TestScope
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+import org.mockito.kotlin.verify
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.analytics.AnalyticsTracker.Stat.DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_SHOWN
+import org.wordpress.android.ui.domains.management.usedomain.UseDomainViewModel.ActionEvent
+import org.wordpress.android.ui.domains.management.usedomain.UseDomainViewModel.ActionEvent.GoBack
+import org.wordpress.android.ui.domains.management.usedomain.UseDomainViewModel.ActionEvent.GoToDomainPurchasing
+import org.wordpress.android.ui.domains.management.usedomain.UseDomainViewModel.ActionEvent.GoToExistingDomain
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+
+@ExperimentalCoroutinesApi
+class UseDomainViewModelTest : BaseUnitTest() {
+    @Mock
+    private lateinit var analyticsTracker: AnalyticsTrackerWrapper
+    private lateinit var viewModel: UseDomainViewModel
+
+    @Before
+    fun setUp() {
+        MockitoAnnotations.openMocks(this)
+        viewModel = UseDomainViewModel(
+            mainDispatcher = testDispatcher(),
+            analyticsTracker = analyticsTracker
+        )
+    }
+
+    @Test
+    fun `WHEN ViewModel initialized THEN track DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_SHOWN event`() {
+        verify(analyticsTracker).track(DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_SHOWN)
+    }
+
+    @Test
+    fun `WHEN new domain selected THEN track DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_NEW_DOMAIN_TAPPED event`() {
+        viewModel.onNewDomainSelected()
+        verify(analyticsTracker).track(DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_SHOWN)
```

Comment:
Thank you for adding tests! Did you perhaps forget to update the verified tracked event in some of these tests? (they're all verifying `DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_SHOWN`).

Reply:
Yeah, I copy-pasted the analytics-related tests and forgot to change the events. 🤦‍♂️
 All the tests pass because `DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_SHOWN` is initially fired event. Already fixed it. Thank you for catching this!